stable/appmesh-controller/README.md

```diff
@@ -86,11 +86,48 @@ helm upgrade -i appmesh-controller eks/appmesh-controller \
 
 The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
-**Note:** When using IRSA, make sure the Envoy proxies have the following IAM policies attached for Envoy to authenticate with AWS App Mesh and fetch it's configuration
+**Note**
+Make sure that the Envoy proxies have the following IAM policies attached for the Envoy to authenticate with AWS App Mesh and fetch it's configuration
 - https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/config/iam/envoy-iam-policy.json
 
-#### Setup IAM permissions manually on worker nodes
-If not setting up IAM role for service account, apply the IAM policies to your worker nodes:
+There are **2 ways** you can attach the above policy to the Envoy Pod  
+#### With IRSA     
+Download the Envoy IAM polocy  
+```
+curl -o envoy-iam-policy.json https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/config/iam/envoy-iam-policy.json
+```
+
+Create an IAM policy called AWSAppMeshEnvoyIAMPolicy
+```
+aws iam create-policy \
+    --policy-name AWSAppMeshEnvoyIAMPolicy \
+    --policy-document file://envoy-iam-policy.json
+```
+
+Take note of the policy ARN that is returned
+
+Create Application namespace to which Envoy proxy will get injected
+```
+kubectl create ns <ApplicationNamespaceName>
+```
+
+Create an IAM role for service account for the App Mesh Envoy, use the ARN from the step above
+```
+eksctl create iamserviceaccount --cluster $CLUSTER_NAME \
+    --namespace <ApplicationNamespaceName> \
+    --name envoy-proxy \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/AWSAppMeshEnvoyIAMPolicy  \
+    --override-existing-serviceaccounts \
+    --approve
+```
+
+Reference this Service Account in your deployment pod spec. This should be the pod which would get injected with the Envoy. Refer below example:
+```
+https://github.com/aws/aws-app-mesh-examples/blob/5a2d04227593d292d52e5e2ca638d808ebed5e70/walkthroughs/howto-k8s-fargate/v1beta2/manifest.yaml.template#L220
+``` 
+
+#### Without IRSA   
+If not setting up IAM role for service account, apply the IAM policies manually to your worker nodes:
```

Comment:
Do we need to add them when not using IRSA? Did we verify that?

Reply:
We already had this in the doc.